src/robusta/core/sinks/robusta/dal/supabase_dal.py

```diff
@@ -293,6 +295,45 @@ def publish_jobs(self, jobs: List[JobInfo]):
             status_code = res.get("status_code")
             raise Exception(f"publish jobs failed. status: {status_code}")
 
+    def remove_deleted_job(self, job : JobInfo):
+        if not job:
+            return
+
+        res = (
+            self.__delete_patch(self.client.table(JOBS_TABLE).delete()
+                .eq("account_id", self.account_id)
+                .eq("cluster_id", self.cluster)
+                .eq("service_key", job.get_service_key())))
+        status_code = res.get("status_code")
+        if status_code != 204:
```

Comment:
I would not limit it to 204. 
I think 200 or 202 is also a valid response for a DELETE operation.
I understand it's what currently returned, but supporting also 200 might be safer.
See here: https://restfulapi.net/http-methods/#:~:text=DELETE%20API%20Response,was%20already%20removed.